### DISCUSSION TITLE
Environment in Liquid

### DISCUSSION THREAD

**Discussion by CChurchill404** [OrchardCMS/OrchardCore#13345] [Q&A]
Hi all, 

Is there a way to detect the current environment using liquid (Development / Production), if there is please let me know

**Comment by Skrypt** [comment]
You will need to extend Liquid to get Environment variables.

**Comment by ns8482e** [comment]
try and see if this works?
```liquid
{% block  name:"environment", names:"Development" %}
...
{%endblock%}
```
